code/events/schedule.tsv

```diff
@@ -1,28 +1,38 @@
 18	2023-10-28	LR	fixation_2023-10-28_19h28.44.805_1_session_18.log	qct_2023-10-28_20h04.55.538_0_session_18.log	rest_2023-10-28_20h14.36.133_0_session_18.log	bht_2023-10-28_20h36.56.464_0_session_18.log
 19	2023-10-29	PA	fixation_2023-10-29_17h27.42.762_0_session_19.log	qct_2023-10-29_18h03.39.805_0_session_19.log	rest_2023-10-29_18h12.09.392_0_session_19.log	bht_2023-10-29_18h33.57.024_0_session_19.log
 20	2023-10-29	RL	fixation_2023-10-29_19h19.52.151_1_session_20.log	qct_2023-10-29_19h59.33.715_0_session_20.log	rest_2023-10-29_20h05.09.929_0_session_20.log	bht_2023-10-29_20h27.21.398_0_session_20.log
-21	2023-10-30	RL	fixation_2023-10-30_19h13.50.101_1_session_21.log	qct_2023-10-30_19h54.05.659_0_session_21.log	rest_2023-10-30_20h02.51.496_0_session_21.log	bht_2023-10-30_20h24.56.078_0_session_21.log
+21	2023-10-30	RL	fixation_2023-10-30_19h14.23.626_2_session_21.log	qct_2023-10-30_19h54.05.659_0_session_21.log	rest_2023-10-30_20h02.51.496_0_session_21.log	bht_2023-10-30_20h24.56.078_0_session_21.log
 22	2023-10-30	AP	fixation_2023-10-30_21h04.29.483_1_session_22.log	qct_2023-10-30_21h42.46.455_0_session_22.log	rest_2023-10-30_21h51.11.079_0_session_22.log	bht_2023-10-30_22h13.00.516_0_session_22.log
 23	2023-10-31	AP	fixation_2023-10-31_19h38.18.771_1_session_23.log	qct_2023-10-31_20h16.23.124_0_session_23.log	rest_2023-10-31_20h23.08.716_0_session_23.log	bht_2023-10-31_20h45.02.641_0_session_23.log
-24	2023-10-31	AP	fixation_2023-10-31_21h38.40.916_1_session_24.log	qct_2023-10-31_22h29.32.491_0_session_24.log	rest_2023-10-31_22h36.04.198_0_session_24.log	bht_2023-10-31_22h58.13.607_0_session_24.log
+24	2023-10-31	AP	fixation_2023-10-31_21h49.38.233_2_session_24.log	qct_2023-10-31_22h29.32.491_0_session_24.log	rest_2023-10-31_22h36.04.198_0_session_24.log	bht_2023-10-31_22h58.13.607_0_session_24.log
 25	2023-11-01	RL	fixation_2023-11-01_19h34.54.237_1_session_25.log	qct_2023-11-01_20h13.26.444_0_session_25.log	rest_2023-11-01_20h19.46.432_0_session_25.log	bht_2023-11-01_20h41.25.152_0_session_25.log
 26	2023-11-01	PA	fixation_2023-11-01_21h19.06.608_1_session_26.log	qct_2023-11-01_21h55.19.742_0_session_26.log	rest_2023-11-01_22h03.59.490_0_session_26.log	bht_2023-11-01_22h26.03.086_0_session_26.log
+27	2023-11-02	AP	fixation_2023-11-02_19h29.37.833_1_session_27.log	n/a	n/a	n/a
 28	2023-11-02	RL	fixation_2023-11-02_21h17.30.822_1_session_28.log	qct_2023-11-02_21h55.55.757_0_session_28.log	rest_2023-11-02_22h03.31.967_0_session_28.log	bht_2023-11-02_22h25.06.651_0_session_28.log
+29	2023-11-03	PA	fixation_2023-11-03_19h15.36.495_2_session_29.log	qct_2023-11-03_19h58.22.540_0_session_29.log	rest_2023-11-03_20h06.12.290_0_session_29.log	bht_2023-11-03_20h27.56.353_0_session_29.log
+30		AP	n/a	n/a	n/a	n/a
+31		PA	n/a	n/a	n/a	n/a
+32		AP	n/a	n/a	n/a	n/a
+33		LR	n/a	n/a	n/a	n/a
+34		LR	n/a	n/a	n/a	n/a
+35		LR	n/a	n/a	n/a	n/a
+36		RL	n/a	n/a	n/a	n/a
```

Comment:
What's the point of this?
```suggestion
```

See https://github.com/TheAxonLab/hcph-sops/blob/mkdocs/code/events/schedule-pilot-excl.tsv